client/src/components/ViolationsSummary.tsx

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+import * as _ from "lodash";
+import { MaybeStringyNumber } from "../util/helpers";
+import { Trans } from "@lingui/macro";
+
+const VIOLATIONS_AVG = 0.7; // By Unit
+// 1656793 open violations according to wow_bldgs
+// 2331139 total units in registered buildings, according to wow_bldgs
+
+export const ViolationsSummary: React.FC<{
+  openviolationsperresunit: MaybeStringyNumber;
+  totalviolations: number;
+}> = props => {
+  const { openviolationsperresunit, totalviolations } = props;
+
+  // openviolationsperresunit is not defined when the portfolio has 0 residentials
+  // units in PLUTO, so we can't calculate a proportion.
+  const openViolations = _.isNil(openviolationsperresunit) ? (
+    <Trans>
+      The average number of open HPD violations per residential unit is not available for this
+      portfolio.
+    </Trans>
```

Comment:
Hmm, I wonder if instead of saying that this statistic is "not available" here, we could just remove this line altogether. In my experience, having HPD-registered buildings with 0 residential units either means there is an error in PLUTO's tally, or there are actually no residential units and the building had to register for some other reason (or was registered at a time when there used to be residential units). Therefore, it may be simpler just to remove this line for portfolios of 0 units, as finding the average by unit just wouldn't make logical sense. Happy to hear your thoughts on this though!

Reply:
That seems fair. I'll remove it.